sdk/cosmos/azure-cosmos-encryption/src/test/java/com/azure/cosmos/encryption/CosmosEncryptionAsyncClientCacheTest.java

```diff
@@ -0,0 +1,52 @@
+package com.azure.cosmos.encryption;
```

Comment:
missing license header file.

Reply:
Deleted that file to move the test case to CosmosEncryptionClientCachesTest.